lib/shared/drive-constraints.js

```diff
@@ -208,3 +208,85 @@ exports.isDriveValid = (drive, image) => {
 exports.isDriveSizeRecommended = (drive, image) => {
   return _.get(drive, [ 'size' ], UNKNOWN_SIZE) >= _.get(image, [ 'recommendedDriveSize' ], UNKNOWN_SIZE);
 };
+
+const feedbackMessages = {
```

Comment:
Even better: what about exporting these, so the client code can check which message is which? This could be really handy to address https://github.com/resin-io/etcher/pull/1193/files#r107220266: this function can determine flashing to a system drive is an error, while the GUI can check if that's the case by comparing with one of these constants, and dismissing the error if the unsafe mode setting is enabled.